flink-runtime-web/web-dashboard/app/scripts/index.coffee

```diff
@@ -30,7 +30,7 @@ angular.module('flinkApp', ['ui.router', 'angularMoment', 'dndLists'])
 
 .value 'flinkConfig', {
   jobServer: ''
-#  jobServer: 'http://localhost:8081/'
+  # jobServer: 'http://localhost:8081/'
```

Comment:
I updated it because I had the same problem with sublime :) ... It was easier for my workflow to indent it like this.